docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -10,10 +10,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_config: configuration/docker.adoc
 :url_FLUSHALL: https://redis.io/commands/FLUSHALL
 
+NOTE: To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and then follow the existing process described below.
```

Comment:
what process are you referring to here exactly?

Reply:
Yes this is a good point Mona (The same as in the source service guide), so I have reworded it accordingly.